Makefile.am

```diff
@@ -166,19 +166,18 @@ BATS = \
 	test/functional/search/search-content-check-negative.bats \
 	test/functional/search/search-content-check-positive.bats \
 	test/functional/search/search-client-certificate.bats \
-	test/functional/update/update-apply-full-file-delta.bats \
```

Comment:
why are these going away? I didn't see any mention of this in your commit message.